manifests/t/TheOSCARTeam/OSCAR/1.3.1.500/TheOSCARTeam.OSCAR.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: TheOSCARTeam.OSCAR
+PackageVersion: 1.3.1.500
+PackageLocale: en-US
+Publisher: The OSCAR Team
+PackageName: OSCAR
+License: Copyright 2019-2020 The OSCAR Team

```

Comment:
A copyright is not a license; Please provide the actual license terms such as `Proprietary`, `MIT`, `GPL-2.0` or other license terms as applicable